### DISCUSSION TITLE
11/21 생각 나누기

### DISCUSSION THREAD

**Discussion by Orm-camp** [Orm-camp/blog-feedback#34] [동글동글 3기 피드백]
![image](https://github.com/user-attachments/assets/023c635d-204a-49b0-beac-2e6dc60d69ce)

오늘의 글을 보시고 든 생각, 의견 자유롭게 댓글로 남겨주세요 🌈

**Comment by structified** [comment]
블로그의 방향성이나 타겟층을 명확하게 설정하는 것이 “꾸준함”에 있어서는 방해 요소가 될수도 있겠다는 생각이 들었습니다.

**Reply by Orm-camp** [reply]
어떤 이유에서 방해 요소라고 생각하셨나요? 더 자세한 이야기를 들어보고 싶어요!

**Reply by structified** [reply]
주제에 대해서 너무 고민하지 말고 쉽고 당연한 글이라도 일단 올리자는 취지의 내용인것 같아서요!

**Comment by baaanjy** [comment]
최신글을 더 신뢰한다는 점 공감합니다!

**Comment by naseulgee** [comment]
공감되는 말인 것 같습니다.
열심히 이것저것 연구해보고 기록을 세세하게 사진찍어 올린적이 있었는데,
노력대비 반응이 없어 몇달동인 블로그 자체를 접은적이 떠오르네요ㅎㅎ

**Comment by ghost** [comment]
글을 작성할 때 가볍고 짧은 글이라도 작성하고 싶은데 그게 잘 안되네요..

**Comment by hansol-woff** [comment]
일단 가볍게 쓰고 반응을 살피고 시간을 투자한다는 개념이 효율면에서도 좋고, 독자와 활발한 상호작용이 있는 블로그라는 인상을 주기도 좋아보입니다.

**Comment by oratio100** [comment]
프로젝트 회고처럼 많은 내용이 들어갈 것 같은 글은 진도가 잘 안나가고 있는데, 초기에는 짧은 글이라도 꾸준히 올리는 것에 집중을 해봐야겠다는 생각이 들었습니다.

**Comment by mh0223** [comment]
이번에 블로그 글을 쓰면서 글이 길어지는 문제가 있었는데 공감이 갑니다,,

**Comment by kimjigyun** [comment]
저도 글을 쓰다보면 길어지는데 .. 모든것에 꾸준함이 필요한듯 합니다.

**Comment by ksj0621** [comment]
저는 오히려 짧은 글보단 상세하고 자세한 정보글을 더 좋아합니다. 초보자의 관점에서 개발 문제들 하나하나가 복잡하고 어려운 부분이 있어서 그런 글들의 도움을 더 많이 받았었기 때문에 제가 정보글을 쓰게 된다면 최대한 상세하게 쓸 것이라고 마음 먹었었습니다,, 짧은 글들에 대한 여론도 생각해 볼 수 있어 좋았습니다!
